packages/wdio-utils/tests/__mocks__/@wdio/utils.ts

```diff
@@ -13,7 +13,9 @@ import {
 } from '../../../src/envDetector'
 
 class DotReporter {
-    constructor (options) {
+    options: any
```

Comment:
Can any be replaced here?

Reply:
not yet, we are waiting for #5909 to land